src/plans/create/steps/other-settings/constants.ts

```diff
@@ -22,3 +24,16 @@ export const defaultDiskPassPhrase: DiskPassPhrase = { value: '' };
 export const defaultTransferNetwork = t('Target provider default');
 
 export const maxDiskPassPhrases = 20;
+
+export type TargetPowerStateValue = 'auto' | 'on' | 'off';
```

Comment:
Please add constants to auto on and off and use them accross this PR, this will also be used in the next steps so consider high level for these type and constants